.travis.yml

```diff
@@ -28,7 +28,7 @@ before_install:
       fi
 
 install:
-    - sh sbin/travis.sh bootstrap
+    - sh sbin/travis.sh bootstrap --with-color-scheme-unit
```

Comment:
may be we should prepare a command like `sh sbin/travis.sh install_color_scheme_unit` instead?

Reply:
If you prefer, sure. It would probably make more sense as `sh sbin/travis.sh install_color_scheme_unit`